core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -393,8 +394,8 @@ public void init(Properties p) throws WorkloadException {
     String scanlengthdistrib =
         p.getProperty(SCAN_LENGTH_DISTRIBUTION_PROPERTY, SCAN_LENGTH_DISTRIBUTION_PROPERTY_DEFAULT);
 
-    int insertstart =
-        Integer.parseInt(p.getProperty(INSERT_START_PROPERTY, INSERT_START_PROPERTY_DEFAULT));
+	int insertstart=Integer.parseInt(p.getProperty(INSERT_START_PROPERTY,INSERT_START_PROPERTY_DEFAULT));
```

Comment:
Retain the original formatting Daniel... there is probably a reason (line length or something).